getting_started.md

```diff
@@ -0,0 +1,104 @@
+
+You don't need to upload an image, but I recommend that you do, because it's much easier to figure out who is talking when there's a visual, and the defaults are really generic. Some people use a photo of themselves. Some people use a random avatar they like, just like Twitter.
+
+The text area, like every text area in Patchwork uses [markdown](http://commonmark.org/help/) formatting.
+
+Save that when you're done.
+
+## Step 3 
+
+Get connected.  
+Because Scuttlebutt is a distributed network you need to find some people to follow. This is partially because it's really _really_ boring if you're not listening to anyone and no-one is listening to you. It's also because you get "gossip" from the people you follow. "Gossip" is more than just info about people you're interested in. It is also the name of [a protocol](https://en.wikipedia.org/wiki/Gossip_protocol) that Scuttlebutt uses to share information. 
+
+### Connecting Locally
+Most of us don't happen to be sitting next to someone else on the network. If you _are_ you'll see them listed under "local" on the left. Click their profile to see more about them, and click "Follow" if you feel like following what that person has to say (just like Twitter).  You'll start downloading information from them that they've posted to the network _and_ information their friends have posted. 
+
+### Remotely
+Even if you _are_ next to someone you'll still want to connect to a "pub" server so that you can exchange info with others if no-one is nearby. In the video they're described as being more like bartenders than pubs, because it's the bartender who you gossip with at the pub, and shares the gossip of others.
+
+Go to [the list of pub servers](https://github.com/ssbc/scuttlebot/wiki/Pub-Servers). 
+
+Open one up in a new tab and you'll see something like this.
+
+![pub server example](assets/pub-server-page-1.png)
+
+Click the button to "Get An Invitation" and it'll generate a custom invitation code just for you. Copy the whole thing. Then click the "+ Join Pub" button in Patchwork. 
+
+![Join Pub button](assets/patchwork-join-pub-button.png)
+
+There's a limit to how far gossip will travel through a network of people, and the bartender at a pub is just another "person" in the chain. Because of this it's good to connect to a few pubs. Also, it spreads the load. 
+
+**Note:** If you've already got a friend on Scuttlebutt, ask them what pub(s) they follow. Otherwise it's possible that you'll be out of range of each other, so to speak. 
+
+The pub's you're currently connected to are listed on the left side if you click on "Public".
+
+## Getting all the Gossip
+
+As soon as you connect to a pub or start following a person Scuttlebutt will start syncing down all the "gossip" they have for you, and pass on any "gossip" you have for them. *The first time you connect to a pub server there will be a lot to download. * So, give it some time.
```

Comment:
'*' formatting problem? or at least it confused the GitHub markdown renderer.

Reply:
weird. i guess it's the lack of space before the first one? dunno. should be valid. Will correct.